it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/trustControllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -201,14 +201,15 @@ class CheckYourAnswersControllerISpec extends AuditEnabledSpecHelper
           }
 
           verifyStoreIdentifiersMatch(testJourneyId, expBody = JsString(SuccessfulMatchKey))
-          verifyCreateBusinessVerificationJourney(expBody = testCreateBusinessVerificationJourneyJson(testSautr, testJourneyId, testTrustsJourneyConfig(businessVerificationCheck = true)))
+          verifyCreateBusinessVerificationJourney(expBody = testCreateBusinessVerificationTrustJourneyJson(testSautr, testJourneyId, testTrustsJourneyConfig(businessVerificationCheck = true)))
           verifyAudit()
         }
       }
+
       "identifier match is SuccessfulMatch (for example all postcodes are the same)" should {
```

Comment:
I think if BV fails in any way the submission service will save the registration status as "not called". Perhaps we should stub and verify this.

Reply:
Richard I think this is a bug ... because we are not doing it at the moment:
 _ <- storageService.storeRegistrationStatus(journeyId, RegistrationNotCalled)
 I need to add it.